[2023-03-20T16:04:45.506+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v02.get_sklearn manual__2023-03-20T16:04:38.731365+00:00 [queued]>
[2023-03-20T16:04:45.523+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python_dependencies_v02.get_sklearn manual__2023-03-20T16:04:38.731365+00:00 [queued]>
[2023-03-20T16:04:45.524+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-20T16:04:45.524+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-03-20T16:04:45.525+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-20T16:04:45.550+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): get_sklearn> on 2023-03-20 16:04:38.731365+00:00
[2023-03-20T16:04:45.555+0000] {standard_task_runner.py:55} INFO - Started process 976 to run task
[2023-03-20T16:04:45.559+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_dependencies_v02', 'get_sklearn', 'manual__2023-03-20T16:04:38.731365+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_python_dependencies.py', '--cfg-path', '/tmp/tmpp261v2s1']
[2023-03-20T16:04:45.562+0000] {standard_task_runner.py:83} INFO - Job 98: Subtask get_sklearn
[2023-03-20T16:04:45.656+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_python_dependencies_v02.get_sklearn manual__2023-03-20T16:04:38.731365+00:00 [running]> on host a1e5df821a6d
[2023-03-20T16:04:45.764+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=ThinhHuynh
AIRFLOW_CTX_DAG_ID=dag_with_python_dependencies_v02
AIRFLOW_CTX_TASK_ID=get_sklearn
AIRFLOW_CTX_EXECUTION_DATE=2023-03-20T16:04:38.731365+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-20T16:04:38.731365+00:00
[2023-03-20T16:04:45.767+0000] {taskinstance.py:1775} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 192, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/dag_with_python_dependencies.py", line 14, in get_sklearn
    import sklearn
ModuleNotFoundError: No module named 'sklearn'
[2023-03-20T16:04:45.785+0000] {taskinstance.py:1326} INFO - Marking task as FAILED. dag_id=dag_with_python_dependencies_v02, task_id=get_sklearn, execution_date=20230320T160438, start_date=20230320T160445, end_date=20230320T160445
[2023-03-20T16:04:45.806+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 98 for task get_sklearn (No module named 'sklearn'; 976)
[2023-03-20T16:04:45.852+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-03-20T16:04:45.904+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
